<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndigentsWindow</class>
 <widget class="QMainWindow" name="IndigentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
 background :  ; 
 border-radius : 60px ; 
}
QLabel
{
 color : #48494B  ; 
}
QLineEdit
{
 background:transparent ; 
border : none ;
color:#48494B ;
font-size : 20px ; 
border-bottom:0.5px solid #717072 ; 
}
QTabWidget::pane {
  top:-40px; 
} 
QTabBar::tab {
background-color: rgb(217, 221, 220);
border-radius : 15px;
border: 2px solid lightgray; 
border-color: rgb(255, 255, 255);
padding: 3px;
} 

QTabBar::tab:selected { 
background-color: rgb(217, 221, 220);
border: 3px solid lightgray; 
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 140, 0);
border-right-color: rgb(255, 140, 0);
margin-bottom: 0px; 

}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1121</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(217, 221, 220);
border-radius : 35px ; </string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_CreateIndigents">
     <attribute name="title">
      <string>Create Indigent</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Indegent</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="dob_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>290</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Date of Birthday : </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dob">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>300</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Sexe : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>530</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>530</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sexe">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>390</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineId">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>131</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Id Indigent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_IndigentManagment">
     <attribute name="title">
      <string>Indigent Managment</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>List Indegents</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chercher_nom">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>190</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>520</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>490</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>221</y>
        <width>481</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>470</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>ID of Indigent</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Sexe : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_modif">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sexe_modif">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="dob_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Date of Birthday : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_modif">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dob_modif">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="tri_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
         </property>
         <property name="text">
          <string>Tri Sexe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tri_3">
         <property name="styleSheet">
          <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
         </property>
         <property name="text">
          <string>Tri Date</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>520</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_image_4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>20</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Image indigent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Statistics">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>91</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_stat">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>731</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>511</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QPushButton" name="insertImage_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>insertion image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_imagePath_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>image path</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_image_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>   l'image selectionné</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectImage_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>selection d'image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_inserer_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id_indigent</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
